_posts/2017-12-22-responsive-images-srcset-and-sizes.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: post
+title:  "Responsive Images in HTML5: Srcset and Sizes"
+date: 2017-12-22 09:11:00 +0200
+categories: css
+author_name: Martin Kreidenweis
+author_url : /author/martinkreidenweis
+author_avatar: martinkreidenweis
+show_avatar : true
+read_time : 4
+show_related_posts: false
+square_related:
+feature_image: posts/2017-12-22-responsive-images-srcset-and-sizes/patricia-serna-colores-a-medida-415257.jpg
+---
+
+A long time ago Eric Portis wrote an [excellent article about "Srcset and sizes"][eric-article]. With funny pictures. 
+I've been told it's too long though. Let me summarize.
+
+## The Problem
+
+We're building a "responsive" web site here, meaning we deliver the same HTML and CSS to mobile and desktop browsers. Using CSS media queries the content is then formatted appropriately to fit the Urlauber's screen size. So the same image will be displayed in different sizes, depending on the viewport size.
```

Comment:
Do we really have to use urlauber? I know it’s this HC thing but for everyone else on planet earth it is plain silly.

Reply:
Maybe “at HolidayCheck” instead of “here”?